Proiect/new-evaluation.html

```diff
@@ -9,11 +9,10 @@
 			<img id="logo" src="logo-v2.png">
```

Comment:
I'm not so sure either to be honest..I'm still having some trouble using git. I'll check.